D/DutyCycles/Compat.toml

```diff
@@ -0,0 +1,2 @@
+[0]
+julia = "1.1-1"
```

Comment:
Doesn't work on Julia 1.0?

Reply:
No, it doesn't work on Julia 1.0 due to extensive use of [`isnothing`](https://docs.julialang.org/en/v1/base/base/#Base.isnothing).